test/models/note_test.rb

```diff
@@ -47,15 +47,6 @@ def test_closed?
     assert_not_predicate create(:note, :status => "open", :closed_at => nil), :closed?
   end
 
-  def test_description
-    comment = create(:note_comment)
-    assert_equal comment.body, comment.note.description
-
-    user = create(:user)
-    comment = create(:note_comment, :author => user)
-    assert_equal comment.body, comment.note.description
-  end
```

Comment:
You dropped `note.description` a few commits before. Why did you keep the tests until this commit?

+ same for `note.author_id` and `author_ip`